include/swift/AST/DiagnosticsSema.def

```diff
@@ -1504,6 +1504,9 @@ ERROR(access_control_extension_open,none,
 ERROR(access_control_open_bad_decl,none,
       "only classes and overridable class members can be declared 'open';"
       " use 'public'", ())
+WARNING(access_control_non_objc_open_member,none,
+      "non-@objc %select{properties|methods}0 in extensions cannot be overriden; use 'public' instead",
```

Comment:
There is a typo (also, suggest using single quotation marks around `@objc` for code voice):

```suggestion
      "non-'@objc' %select{properties|methods}0 in extensions cannot be overridden; use 'public' instead",
```